cob_hardware_config/raw3-3/config/robot_modules.yaml

```diff
@@ -1 +1 @@
-robot_modules: ['base','arm']
+robot_modules: ['base','torso']
```

Comment:
What is the torso on raw3-3? The linear actuators on top? Can this be moved via teleop (this is what `robot_modules.yaml` is for, right?) or why do we need to add this here?